sheeprl/algos/dreamer_v2/agent.py

```diff
@@ -278,7 +278,9 @@ def __init__(
             norm_layer=[nn.LayerNorm] if layer_norm else None,
             norm_args=[{"normalized_shape": dense_units}] if layer_norm else None,
         )
-        self.rnn = LayerNormGRUCell(dense_units, recurrent_state_size, bias=True, batch_first=False, layer_norm=True)
+        self.rnn = LayerNormGRUCell(
```

Comment:
Why we don't let the user choose if he/she wants to use the upcasted 32bit version of the LayerNorm?

Reply:
I would open a new branch to allow the user to choose which level standard to use. What do you think?

Reply:
LGTM